test/e2e.sh

```diff
@@ -39,10 +39,10 @@ runWebserver 8090
 WS_EXT_PID=$PID
 
 echo ""
-echo "Fiddle App"
-cd $BASE_DIR/../website/fiddle
-killWebserver 8080
```

Comment:
If we are getting rid of 8080, should we maybe then also replace it [here](https://github.com/SAP/luigi/blob/master/test/e2e-test-application/src/luigi-config/extended/userSettings.js#L92) and [here](https://github.com/SAP/luigi/blob/master/test/e2e-test-application/src/luigi-config/extended/userSettings.js#L105) with the new 4500?

Reply:
Apparently all the tests are green now. But only the usersettings one related to theming is failing. It seems that if you apply the change I mention above it should fix that test as well.